dspace-solr/pom.xml

```diff
@@ -250,9 +250,14 @@
             <scope>runtime</scope>
         </dependency>
         <dependency>
-            <artifactId>log4j</artifactId>
-            <groupId>log4j</groupId>
-            <type>jar</type>
```

Comment:
Why does solr have the type describe as `jar` but no other POM has this?

Reply:
That is a good question, since 'jar' is the default value of <type>.  I would remove it.